content/docs/reference/pkg/python/pulumi/_index.md

```diff
@@ -1198,7 +1198,848 @@ with a resource and stream_id if provided.</p>
 </dd></dl>
 
 </div>
-<div class="section" id="automation-api">
-<h2>Automation API<a class="headerlink" href="#automation-api" title="Permalink to this headline"></a></h2>
+<div class="section" id="module-pulumi.automation">
+<span id="automation-api"></span><h2>Automation API<a class="headerlink" href="#module-pulumi.automation" title="Permalink to this headline"></a></h2>
+<p>The automation module contains the Pulumi Automation API, the programmatic interface for driving Pulumi programs
+without the CLI.
+Generally this can be thought of as encapsulating the functionality of the CLI (<code class="docutils literal notranslate"><span class="pre">pulumi</span> <span class="pre">up</span></code>, <code class="docutils literal notranslate"><span class="pre">pulumi</span> <span class="pre">preview</span></code>,
+<code class="docutils literal notranslate"><span class="pre">pulumi</span> <span class="pre">destroy</span></code>, <code class="docutils literal notranslate"><span class="pre">pulumi</span> <span class="pre">stack</span> <span class="pre">init</span></code>, etc.) but with more flexibility. This still requires a
+CLI binary to be installed and available on your $PATH. The Automation API is in Alpha (experimental package/x)
```

Comment:
Feels like we should probably hold off until https://github.com/pulumi/pulumi/pull/6828 is merged and we can regen these docs without the Alpha text in there.

Reply:
Fixed